File: JanperCab/5-JanperCab/ClientApp/src/app/_models/pick-up-sheet/PickUpSheetForListDto.ts
import { EnquiryForPickUpSheetDto } from './../enquiry/EnquiryForPickUpSheetDto';
import { Type } from 'class-transformer';
import { PickUpSheetDto } from './PickUpSheetDto';

export class PickUpSheetForListDto extends PickUpSheetDto {
  @Type(() => EnquiryForPickUpSheetDto)
  enquiriesForPickUpSheet: EnquiryForPickUpSheetDto[];
}

File: JanperCab/3-Application/Dtos/PickUpSheet/PickUpSheetForListDto.cs
using _3_Application.Dtos.Enquiry;
using System.Collections.Generic;

namespace _3_Application.Dtos.PickUpSheet
{
    public class PickUpSheetForListDto : PickUpSheetDto
    {
        public List<EnquiryForPickUpSheetDto> EnquiriesForPickUpSheet { get; set; }
    }
}